spec/outputs/loggly_spec.rb

```diff
@@ -1 +1,60 @@
+# encoding: utf-8
 require "logstash/devutils/rspec/spec_helper"
+require "logstash/outputs/loggly"
+
+describe 'outputs/loggly' do
+  let(:config) { { "key" => '34567d8-2b67-567-fghj-6789' } }
```

Comment:
This is a fake key but I see how it can be misleading. I'll switch it to something a bit more made up.